introduction.md

```diff
@@ -2,4 +2,5 @@
 *HTML5
 *CSS
 *Python
-*Django
\ No newline at end of file
+*Django
+*loco
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?